scripts/build.js

```diff
@@ -50,63 +49,62 @@ if (POSTINSTALL_BUILD_CWD !== CWD) {
       // Fetch package.json
       var pkgJson = require(path.join(CWD, "package.json"));
       var devDeps = pkgJson.devDependencies;
-      // Values listed under `buildDependencies` contain the dependency names
-      // that are required for `lib` building.
-      var buildDependencies = _.pick(devDeps, pkgJson.buildDependencies);
-
-      // Proceed only if there is something to install
-      if (!_.isEmpty(buildDependencies)) {
```

Comment:
Removed this conditional since this script does not make sense if this array is empty or non existent. AFAIK the `buildDependencies` property on `package.json` is only used by this script, therefore if removed in the future, so does this script.